deployment.yaml

```diff
@@ -19,7 +19,7 @@ spec:
       serviceAccountName: heimdall
       containers:
       - name: heimdall
-        image: quay.io/uswitch/heimdall:v0.5.2
+        image: quay.io/uswitch/heimdall:v0.5.3
```

Comment:
This definitely won't be a patch release. I think we should do `v0.6.0` for this.